packages/preview-middleware/README.md

```diff
@@ -61,15 +58,15 @@ server:
 ```
 
 ### Different Path and Debugging enabled
-With this configuration, the app will be local FLP will be available at `/preview.html` and the log level is `debug`.
+With this configuration, the local FLP will be available at `/test/myFLP.html` and the log level is `debug`.
 ```Yaml
 server:
   customMiddleware:
   - name: preview-middleware
     afterMiddleware: compression
     configuration:
       flp: 
-        path: /preview.html
+        path: /test/myFLP.html
```

Comment:
Ok, a different url, fine, but why is this required?

Reply:
Because `preview.html` is more or less currently taken. I also would like to emphasise that best practice is to have the HTML files in the `test` folder.